libcxx/utils/ci/docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
       dockerfile: Dockerfile
       target: actions-builder
       args:
-        BASE_IMAGE: ghcr.io/actions/actions-runner:2.322.0
+        BASE_IMAGE: ghcr.io/actions/actions-runner:2.326.0
```

Comment:
Should we maybe just use "latest" here?

I don't feel too strongly one way or the other. One approach makes the container build more deterministic but the other requires less maintenance.

Reply:
I'd prefer to keep the version pinned just for reproducibility. I'll probably change how this is wired up at some point too to make upgrading the runner version less of a pain (so that we only touch the runner and not everything else).